CHANGELOG.md

```
@@ -1106,8 +1106,8 @@
 - 3ca9190: Renamed `FallbackSettings.fallbackPolicy` to `FallbackSettings.errorPolicy`
 - 3ca9190: - Removed the following types:
 
-                                                                        - `AsyncFactoryable`
-                                                                        - `Factoryable`
+                                                                                                                  - `AsyncFactoryable`
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)